apps/omg_performance/lib/omg_performance/byzantine_events.ex

```diff
@@ -0,0 +1,121 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Performance.ByzantineEvents do
+  @moduledoc """
+  OMG network child chain server byzantine event test. Setup and runs performance byzantine tests.
+  """
+
+  require OMG.Utxo
+
+  alias OMG.Eth
+  alias OMG.Eth.RootChain
+  alias OMG.State.Transaction
+  alias OMG.Utxo
+  alias OMG.Watcher.HttpRPC.Client
+  alias OMG.Watcher.TestHelper
+
+  def start_dos_get_exits(dos_users, positions) do
+    Enum.map(1..dos_users, fn _ -> worker_dos_exit(positions) end)
+    |> Enum.map(&Task.await(&1, :infinity))
+  end
+
+  defp worker_dos_exit(exit_positions) do
+    worker = fn exit_positions ->
+      exit_positions
+      |> Enum.map(fn position ->
```

Comment:
there's no chaining here so write the mapping like
`Enum.map(exit_positions, ...`